instrumentation/liberty/liberty/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/liberty/ThreadLocalContext.java

```diff
@@ -8,18 +8,21 @@
 import io.opentelemetry.context.Context;
 import io.opentelemetry.context.Scope;
 import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 
 public class ThreadLocalContext {
 
   private static final ThreadLocal<ThreadLocalContext> local = new ThreadLocal<>();
 
   private final HttpServletRequest req;
+  private final HttpServletResponse response;
   private Context context;
   private Scope scope;
   private boolean started;
 
-  private ThreadLocalContext(HttpServletRequest req) {
+  private ThreadLocalContext(HttpServletRequest req, HttpServletResponse response) {
```

Comment:
could you rename `req` -> `request` in this class to make it consistent with our style of using verbose names

Reply:
Done.